package.json

```diff
@@ -5,11 +5,13 @@
   "dependencies": {
     "axios": "1.4.0",
     "classnames": "2.3.2",
+    "lodash": "4.17.21",
```

Comment:
`lodash` is a pretty heavy package for the client to download(not sure about the gzipped size)

<img width="143" alt="image" src="https://github.com/manh-t/react-survey/assets/39977805/d2ec8442-e515-4228-815d-8fdedd09f8cb">

There are two packages that I'd suggest that'd do the same work for you with less code and less downloading overhead for the browser:

- [camelcase-keys](https://www.npmjs.com/package/camelcase-keys)
- [snakecase-keys](https://www.npmjs.com/package/snakecase-keys)

Both the above packages are 1000 times lighter than `lodash`(kB vs mB), also you don't need to do any key transformations...so less code! 💭

Reply:
@tyrro thanks, but I need the other util methods from `lodash` like `isEmpty`. And @nvminhtue suggested me not toi import the whole package `lodash`, but only the one we need like `import cameCase from 'lodash/camelCase';` to reduce the package size. So I think it's fine to keep this package 😄